### ISSUE TITLE
Add points contained info to stats csv

### ISSUE THREAD

**Issue by sweeney-l** [special-uor/ageR#45]
In the outputed csv with stats about the different scenarios, is is possible to have something like no. of points contained within confidence interval as per rbacon: Bacon?

**Comment by villegar** [comment]
> In the outputed csv with stats about the different scenarios, is it possible to have something like no. of points contained within confidence interval as per rbacon: Bacon?

@sweeney-l when you say "points", do you mean "dates"? So for the plot below, you want the stats file to report that all four dates are withing the confidence interval?
<img width="2142" alt="image" src="https://user-images.githubusercontent.com/5126692/102357299-171f9980-3fa6-11eb-95d7-0697acf3e9de.png">

**Comment by sweeney-l** [comment]
Yep, sorry, mean dates. When you run bacon from rbacon you get something that says e.g. "92% of the dates overlap with the age-depth model (95% ranges)". In your example you would get 100% of dates overlap. As it is already in rbacon, I thought it might also be a relatively easy (!?) way of choosing the best model, based on the discussions we had before. It would certainly discount those scenarios where no dates are inside the CI but are still chosen by the package as the best because the abc and bias values are good

**Comment by villegar** [comment]
The only problem here is that since we are running multiple instances of `rbacon::Bacon` in parallel and capturing all those messages in a single log file (see below), I cannot tell which message belongs to a particular scenario. A possible workaround might be to capture individual log files for each scenario, but I have yet to figure out a feasible way to do this.

Example log file:
```bash
100% of the dates overlap with the age-depth model (95% ranges)


--------------------------------------------------------------------------------
|                                Saving results                                |
--------------------------------------------------------------------------------

100% of the dates overlap with the age-depth model (95% ranges)



Mean 95% confidence ranges 406 yr, min. 115 yr at 15.1515151515152 cm, max. 884 yr at 500 cm
--------------------------------------------------------------------------------
|                                Saving results                                |
--------------------------------------------------------------------------------

80% of the dates overlap with the age-depth model (95% ranges)


--------------------------------------------------------------------------------
|                                Saving results                                |
--------------------------------------------------------------------------------
```

Anyhow, I will have a look, as it will definitely help to drop "false positives" for best models.

**Comment by sweeney-l** [comment]
Hmm, yes, I understand.
